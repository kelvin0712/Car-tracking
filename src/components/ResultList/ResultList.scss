.card-container {
    display: flex;
    flex-direction: column;
    border: 1px solid grey;
    border-radius: 5px;
    padding: 10px;
    cursor: pointer;
    -moz-osx-font-smoothing: grayscale;
    backface-visibility: hidden;
    transform: translateZ(0);
    transition: transform 0.25s ease-out;
    margin-top:.5rem;
    margin-right: .5rem;
  }
  
  .card-container:hover {
    transform: scale(1.05);
  }