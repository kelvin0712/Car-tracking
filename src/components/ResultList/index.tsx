import React from 'react';
import { HistoryRecord } from '../../api';

/**
 * Display list of history events
 */
export default (props: {
  historyRecords: HistoryRecord[]
}) => <div>
  {props.historyRecords.map(record =>
    <div key={record.id}>
      <p> Driver Name: {record.firstName}</p>
    </div>
  )}
</div>
