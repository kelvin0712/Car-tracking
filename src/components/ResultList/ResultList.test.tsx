import React from 'react';
import { render } from '@testing-library/react';
import ResultList from '.';

it('Should match shapshot', () => {
  const { asFragment } = render(<ResultList historyRecords={[
    { firstName: 'Test' }
  ]} />)

  expect(asFragment()).toMatchSnapshot()
})

