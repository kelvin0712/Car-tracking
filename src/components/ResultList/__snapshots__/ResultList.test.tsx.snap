// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match shapshot 1`] = `
<DocumentFragment>
  <div>
    <div>
      <p>

         Driver Name: Test
      </p>
    </div>
  </div>
</DocumentFragment>
`;
